pkg/provider/resource_cluster.go

```diff
@@ -35,47 +35,47 @@ func (c *ClusterResource) Schema() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"allowed_ip_ranges": {
-				Description: "Allowed IP ranges",
+				Description: "Allowed IP ranges.",
 				Type:        schema.TypeList,
 				Optional:    true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"cidr_block": {
-							Description: "CIDR Block",
+							Description: "CIDR Block.",
 							Type:        schema.TypeString,
 							Required:    true,
 						},
 						"description": {
-							Description: "CIDR Block Description",
+							Description: "CIDR Block Description.",
 							Type:        schema.TypeString,
 							Optional:    true,
 						},
 					},
 				},
 			},
 			"backup_retention_period": {
-				Description: "Backup Retention Period.",
+				Description: "Backup Retention Period. e.g. \"7d\", \"2w\" or \"3m\".",
```

Comment:
```suggestion
				Description: "Backup Retention Period. For example, \"7d\", \"2w\", or \"3m\".",
```

Reply:
e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
serial comma, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/a7e9930ede32c6adc1fc0810c56a60d8391f3fb3